src/consts/AppConfig.ts

```diff
@@ -6,5 +6,5 @@ export const AppConfig = {
   XMTP_ENV: 'dev' as 'local' | 'dev' | 'production',
   MULTI_WALLET: false,
   PUSH_NOTIFICATIONS: Platform.OS === 'ios',
-  GROUP_CONSENT: false,
+  GROUP_CONSENT: Platform.OS === 'android',
```

Comment:
why just android?

Reply:
Bug on the iOS side: https://github.com/xmtp/xmtp-react-native/issues/391